python/lsst/ts/MTDome/encoding_tools.py

```diff
@@ -21,33 +21,34 @@
 
 import logging
 import json
-import pathlib
 
 import jsonschema
 
+from .schema import (
+    AMCS_STATUS_JSCHEMA,
+    APSCS_STATUS_JSCHEMA,
+    COMMAND_JSCHEMA,
+    LCS_STATUS_JSCHEMA,
+    LWSCS_STATUS_JSCHEMA,
+    MONCS_STATUS_JSCHEMA,
+    RESPONSE_JSCHEMA,
+    THCS_STATUS_JSCHEMA,
+)
```

Comment:
I am a bit unhappy about the repetition and the unclear connection between the schema constant names and the names by which you refer to the schema in your code. I'm not sure it's worth improving, but here are two ideas:

Make a schema registry (a dict of name: schema). This is a common and simple solution that scales well to adding new schemas. For instance add a new module `schemas/registry.py`:
```
__all__ = ["registry"]
import typing
registry: typing.Dict[str, str] = {}
```

Then each schema module is tweaked to look like this (notice that the names like AMCS_STATUS_JSHEMA are no longer needed, which is a good sign):
```
__all__ = []
import json
from .registry import registry
registry["AMCS"] = json.loads(
...
)
```
and `schemas/__init__.py` looks the same except you append `from .registry import *` to get the only public symbol from the schema sub-package. Note that it still must import all the schema modules _even though they have no public symbols_ in order to add the schemas to the registry.

A slightly fancier variant is to have a `register_schema` function that does some checking, such as rejecting duplicate names and perhaps even validating the schema, before adding the schema to the registry.

Reply:
It turns out that Mypy doesn't like the `__all__ = []` in the schema code files, so I removed those. Apart from that the simple solution works like a charm. Thanks very much.

Reply:
Without an `__all__` every symbols is imported. I think all mypy is looking for is:
```
__all__: typing.List[str] = []
```